src/OptOut.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\OptOut.
+ */
+
+namespace Netzstrategen\CoreStandards;
+
+class OptOut {
+
+  /**
+   * Default Google Analytics opt out script.
+   *
+   * @var string
+   */
+  private static $opt_out_script = <<<CODE
+<script>
+// Disable tracking if the opt-out cookie exists.
+for (var i = 0; i < window.gaIdList.length; i++) {
+  var disableStr = 'ga-disable-' + window.gaIdList[i];
+  if (document.cookie.indexOf(disableStr + '=true') > -1) {
+    window[disableStr] = true;
+  }
+}
+/**
+ * Sets cookies to indicate Google Analytics tracking should be disabled.
+ */
+function gaOptout() {
+  var disableStr;
+  for (var i = 0; i < window.gaIdList.length; i++) {
+    disableStr = 'ga-disable-' + window.gaIdList[i];
+    document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';
+    window[disableStr] = true;
+  }
+}
+</script>
+
+CODE;
+
+  /**
+   * @implements admin_init
+   */
+  public static function admin_init() {
+    // Register opt-out related settings.
+    register_setting(Plugin::PREFIX . '-settings', 'core_standards_optout_ids');
+    register_setting(Plugin::PREFIX . '-settings', 'core_standards_optout_script');
+    register_setting(Plugin::PREFIX . '-settings', 'core_standards_optout_active');
+
+    add_action('admin_enqueue_scripts', __CLASS__ . '::admin_enqueue_scripts');
+  }
+
+  /**
+   * @implements init
+   */
+  public static function init() {
+    if (is_admin()) {
+      return;
```

Comment:
This check should be performed before registering the callback instead of here, so that the class/callback is not loaded in situations where it isn't needed.